android/clientmanager/src/main/java/io/mosip/registration/clientmanager/dao/RegistrationDao.java

```diff
@@ -40,4 +40,10 @@ public interface RegistrationDao {
     @Insert
     void insert(Registration registration);
 
+    @Query("SELECT * FROM registration where client_status = :status order by cr_dtimes desc")
+    List<Registration> findAllRegistrationByStatus(String status);
```

Comment:
we are displaying count, we should rather get count from db itself

Reply:
I have made changes